content/posts/2025-03-10-deepseek-r1.md

```diff
@@ -0,0 +1,221 @@
+---
+date: '2025-03-09T09:30:00-08:00'
+draft: false
+title: 'DeepSeek-R1 671B multi-host Deployment in AIBrix'
+disableShare: true
+hideSummary: true
+searchHidden: false
+ShowReadingTime: false
+ShowWordCount: false
+ShowBreadCrumbs: true
+ShowPostNavLinks: true
+ShowRssButtonInSectionTermList: false
+UseHugoToc: true
+ShowToc: true
+tocopen: true
+---
+
+This blog post introduce deploying DeepSeek R1 using AIBrix. DeepSeek-R1 demonstrates remarkable proficiency in reasoning tasks through step-by-step training process.It features 671B total parameters with 37B active parameters, and 128k context length. However, due to its 671B model size, the deployment process is a bit more complicated. AIBrix provides enough tools that enables users to deploy and manage distributed inference services efficiently.
+
+![deepseek-r1](/images/deepseek-r1/deepseek-performance.jpg)
+ref: https://huggingface.co/deepseek-ai/DeepSeek-R1/resolve/main/figures/benchmark.jpg
+
+## Prerequisite
+
+Before the AIBrix deployment, some preliminary work such as model weight download to object storage or share file system, and customized container image setup has been completed. We don't cover all the steps but just address the critical part instead. You can check our [code samples and tutorial](https://github.com/vllm-project/aibrix/tree/main/samples/deepseek-r1) for more details.
```

Comment:
Line 25, suggested change

"Before deploying DeepSeek-R1 in AIBrix, some preliminary tasks such as downloading model weights to object storage or a shared file system and setting up a customized container image must be completed. This blog will focus on the critical steps rather than covering all details. You can check our [code samples and tutorial](https://github.com/vllm-project/aibrix/tree/main/samples/deepseek-r1) for more details."

Reply:
done